data_steward/analytics/cdr_ops/combined.py

```diff
@@ -0,0 +1,479 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.7.1
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# ## Notebook parameters
+
+# + tags=["parameters"]
+PROJECT_ID = ''  # identifies the project containing the datasets
+DATASET_ID = ''  # the dataset to evaluate
+BASELINE_DATASET_ID = ''  # a baseline dataset for metrics comparison
```

Comment:
From @rfrancis1 : Add a comment saying that this should be an older combined dataset